plugins/inputs/windows_event_log/wineventlog/utils.go

```diff
@@ -99,6 +111,28 @@ func UTF16ToUTF8Bytes(in []byte, length uint32) ([]byte, error) {
 	return decoded, err
 }
 
+func IsTheEndOfContent(in []byte, length uint32) bool {
```

Comment:
I know that this function is implicitly tested via the outer function and the new unit test. 

That being said, is there a specific reason this function is made public (IIRC capitalizing it makes it accessible through the package)? If we're exposing this I think it warrants adding unit tests for this specific function. Otherwise, I think we should make it private scoped

Reply:
Good call, leaving it private make more sense.